src/f32-vbinary/gen/f32-vdiv-minmax-rvv-u8v.c

```diff
@@ -0,0 +1,48 @@
+// Auto-generated file. Do not edit!
+//   Template: src/f32-vbinary/vop-rvv.c.in
+//   Generator: tools/xngen
+//
+// Copyright 2023 SiFive, Inc.
```

Comment:
Update to 2024 if this is the first time the code lands

Reply:
Done!